app/helpers/import_gpx.rb

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+require 'indoor_helper'
+require 'optparse'
+require 'building'
+require 'nokogiri'
+
+include IndoorHelper
```

Comment:
I moved the logic for creating rooms and points to the indoor helper, because I think it's important this import pipeline is tested for future adjustments (I added tests in indoor_helper_spec.rb). Rubocop does not like top level includes. If you know how to import a method from IndoorHelper without include please adjust this. I would be fine with ignoring this for this particular script.

Reply:
If IndoorHelper is a class instead of a module we don't need the include. But it doesn't really make sense for the helper to be a class this would just be a workaround to avoid the rubocop error. I wouldn't mind leaving it the way it is.